packages/core/src/generators/init/generator.ts

```diff
@@ -119,3 +129,20 @@ function addPrepareScript(host: Tree) {
   packageJson.scripts.prepare = prepareSteps.join(' && ');
   writeJson(host, 'package.json', packageJson);
 }
+
+function initBuildCustomization(host: Tree) {
+  const initialized = host.exists('Directory.Build.props');
+  if (!initialized && !isDryRun()) {
+    const checkModuleBoundariesScriptPath = normalize(
+      relative(
+        host.root,
+        resolve('@nx-dotnet/core/src/tasks/check-module-boundaries'),
+      ),
+    );
+
+    generateFiles(host, path.join(__dirname, 'templates/root'), '.', {
+      tmpl: '',
+      checkModuleBoundariesScriptPath,
+    });
+  }
```

Comment:
This doesn't need the isDryRun flag, it is only necessary when the dot net cli is creating files. When we generate files through nx devkit, nx handles dry run for us